src/vuln_analysis/configs/config-ttc.yml

```
@@ -0,0 +1,184 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+general:
+  use_uvloop: true
+
+functions:
+  cve_generate_vdbs:
+    _type: cve_generate_vdbs
+    agent_name: cve_agent_executor  # Used to determine which tools are enabled
+    embedder_name: nim_embedder
+    base_git_dir: .cache/am_cache/git
+    base_vdb_dir: .cache/am_cache/vdb
+    base_code_index_dir: .cache/am_cache/code_index
+  cve_fetch_intel:
+    _type: cve_fetch_intel
+  cve_process_sbom:
+    _type: cve_process_sbom
+  cve_check_vuln_deps :
+    _type: cve_check_vuln_deps
+    skip: false
+  cve_checklist:
+    _type: cve_checklist
+    llm_name: checklist_llm
+  Container Image Code QA System:
+    _type: local_vdb_retriever
+    embedder_name: nim_embedder
+    llm_name: code_vdb_retriever_llm
+    vdb_type: code
+    return_source_documents: false
+  Container Image Developer Guide QA System:
+    _type: local_vdb_retriever
+    embedder_name: nim_embedder
+    llm_name: doc_vdb_retriever_llm
+    vdb_type: doc
+    return_source_documents: false
+  Lexical Search Container Image Code QA System:
+    _type: lexical_code_search
+    top_k: 5
+  Internet Search:
+     _type: serp_wrapper
+     max_retries: 5
+  cve_agent_executor:
+    _type: cve_agent_executor
+    llm_name: cve_agent_executor_llm
+    tool_names:
+      - Container Image Code QA System
+      - Container Image Developer Guide QA System
+      # - Lexical Search Container Image Code QA System  # Uncomment to enable lexical search
+      - Internet Search
+    max_concurrency: null
+    max_iterations: 10
+    prompt_examples: false
+    replace_exceptions: true
+    replace_exceptions_value: "I do not have a definitive answer for this checklist item."
+    return_intermediate_steps: false
+    verbose: false
+  cve_summarize:
+    _type: cve_summarize
+    llm_name: summarize_llm
+  cve_justify:
+    _type: cve_justify
+    llm_name: justify_llm
+  cve_file_output:
+    _type: cve_file_output
+    file_path: .tmp/ttc_output.json
```

Minor: move `ttc` to suffix
```suggestion
    file_path: .tmp/output_ttc.json
```

---

I did that for my own convenience. Didn't think it made it into the shipped config. Removing it now.